app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,16 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
+
+  def map
+    code_schools = CodeSchools.new.all
+    @lat_longs = []
+
+    code_schools.each do |hash| 
```

Comment:
I know it's a limited use variable but I think `school` would be more descriptive here than `hash`.